docs/HPCCMonitoring/HPCCMonitoringAndReporting.xml

```diff
@@ -1191,6 +1190,17 @@
                   </row>
 
                   <row>
+                    <entry>-set_esp_username_pw</entry>
+
+                    <entry>Set specific logins credentials for esp checks. All
+                    fields are required. (esp name, user name, password) Can
+                    be specified more than once.</entry>
```

Comment:
Set specific login credentials for ESP checks. All fields are required (esp name, user name, password). Can
be specified more than once to support multiple ESP servers.